src/plantGlucoseSimulation.ts

```diff
@@ -112,6 +113,18 @@ export class PlantGlucoseSimulation {
     });
   }
 
+  private addInitialGlucosesToStorage() {
+    const realAnimationDuration = this.animationDuration;
+    this.animationDuration = 500;
+    for (let i = 0; i < this.settings.initialGlucoseStored; i++) {
+      let glucose: GlucoseToStorage;
+      glucose = new GlucoseToStorage1(this);
```

Comment:
Use const, write in one line?
```suggestion
      const glucose = new GlucoseToStorage1(this);
```